website/docs/configuration.md

```diff
@@ -66,15 +125,15 @@ module.exports = {
   // ...
   plugins: [
     [
-      '@docusaurus/plugin-content-blog',
```

Comment:
just thinking out loud but maybe we'll want in a future to just be able to use "blog"?

The "content-blog" thing is a bit weird for the users IMHO

Reply:
Quite similarly I'm not fan of the complexity of our i18n paths, I wish we could just have `i18n/fr/blog`